budgetportal/webflow/urls.py

```diff
@@ -11,15 +12,25 @@
 )
 
 urlpatterns = [
-    url(
-        r"^infrastructure-projects/provincial/$",
+    path(
+        r"infrastructure-projects/provincial/",
```

Comment:
out of curiosity, why relax this?

Reply:
do you mean why I removed ^ and $? Its now by default in path.

Reply:
ah cool - thanks - and thanks for updating the others with slug:slug for consistency